modules/mockk-android/src/androidTest/java/io/mockk/Issue1284Test.kt

```diff
@@ -0,0 +1,23 @@
+package io.mockk
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Test
+import org.junit.runner.RunWith
+import kotlin.test.assertEquals
+
+@RunWith(AndroidJUnit4::class)
+class Issue1284Test {
```

Comment:
Can you please rename this test (and its file) to a name that describes the behavior being tested?

We used to have test named after Github issues in the past and went through a huge effort to rename them so that people don't have to go to Github to understand what a test does.

Reply:
OK. Previously, I read the CONTRIBUTING.md file. Maybe this file should be updated now.